terraform.tfvars.json.sample

```diff
@@ -1,25 +1,24 @@
 {
-    "name_prefix": "<YOUR UNIQUE PREFIX NAME TO BE CREATED>",
-    "dns_provider": "azure",
-    "tsb_fqdn": "<YOUR UNIQUE PREFIX NAME TO BE CREATED>.azure.sandbox.tetrate.io",
-    "tsb_version": "1.6.0",
-    "tsb_image_sync_username": "<TSB_REPO_USERNAME>",
-    "tsb_image_sync_apikey": "<TSB_REPO_APIKEY>",
-    "tsb_password": "Tetrate123",
-    "tsb_mp": {
-        "cloud": "azure",
-        "cluster_id": 0
-    },
-    "tsb_org": "tetrate",
-    "aws_k8s_regions": [
-    ],
+    "aws_k8s_regions": [],
```

Comment:
need to align to something like 
```
{
    "name_prefix": "<YOUR UNIQUE PREFIX NAME TO BE CREATED>",
    "tetrate": {
        "version": "1.7.0+tse",
        "image_sync_username": "<TETRATE_REPO_USERNAME>",
        "image_sync_apikey": "<TETRATE_REPO_APIKEY>"
    },
    "k8s_clusters": {
        "aws": [
        {
            "name": "demo1",
            "region": "us-west-1",
            "version": "1.24",
            "management_plane": "true", // only one cluster can be management plane
            "tier1": "true",  
            "external_dns_zone": "aws-ce.sandbox.tetrate.io"
        },
```
https://github.com/tetrateio/tetrate-service-express-sandbox/blob/main/terraform.tfvars.json.advanced.sample

Reply:
This will be done later. First move over to shell scripts.